src/types.d.ts

```diff
@@ -0,0 +1,85 @@
+import type { MongoError } from './error';
+import type { SerializeOptions } from 'bson';
+import type { MongoClient } from '.';
+
+export type UniversalError = MongoError | Error;
```

Comment:
```suggestion
export type AnyError = MongoError | Error;
```
nitpick, but I think it's more "TypeScripty"